eng/common-variables.yml

```diff
@@ -0,0 +1,39 @@
+variables:
+  # Cannot use key:value syntax in root defined variables
+  - name: _TeamName
+    value: DotNetCore
+  - name: _DotNetArtifactsCategory
+    value: .NETCore
+  - name: _DotNetValidationArtifactsCategory
+    value: .NETCoreValidation
+  - name: HelixApiAccessToken
+    value: ''
+  - name: _RunAsPublic
+    value: True
+  - name: _RunAsInternal
+    value: False
+  - name: _InternalBuildArgs
+    value: ''
+
+  - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+    - name: _RunAsPublic
+      value: False
+    - name: _RunAsInternal
+      value: True
+    - name: _SignType
+      value: real
+    # DotNet-Blob-Feed provides: dotnetfeed-storage-access-key-1
+    # Publish-Build-Assets provides: MaestroAccessToken, BotAccount-dotnet-maestro-bot-PAT
+    # DotNet-HelixApi-Access provides: HelixApiAccessToken
+    - group: DotNet-Blob-Feed
+    - group: Publish-Build-Assets
+    - group: DotNet-HelixApi-Access
+    - group: SDL_Settings
+    - name: _InternalBuildArgs
+      value: /p:DotNetSignType=$(_SignType) /p:TeamName=$(_TeamName)
+        /p:DotNetPublishBlobFeedKey=$(dotnetfeed-storage-access-key-1)
+        /p:DotNetPublishBlobFeedUrl=https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
```

Comment:
Do we want corefxlab's packages to be published to this blob feed? I don't think we mix corefxlab packages with our official packages, do we?

Reply:
I think this file is not even imported and we can just delete it since it is confusing. This is the real blob feed we're publishing to: https://github.com/dotnet/corefxlab/pull/2795/files#diff-4eda78ea0bc35794fd331e0d76b7872fR40

we created an experimental one in order to avoid mixing corefxlab with official packages.